.idems_app/deployments/plh/global.config.ts

```diff
@@ -1,15 +1,16 @@
 import { IDeploymentConfig } from "data-models";
-import { getDefaultAppConstants } from "data-models/constants";
+import { getDefaultAppConfig } from "data-models/constants";
+import * as SKINS from "./skins";
```

Comment:
This looks much nicer to me thanks, at some point in the future we could have some global skins in the main folders but for now I think makes a lot more sense to just link with the deployments and copy/paste code as required